filebeat/docs/inputs/input-log.asciidoc

```diff
@@ -57,6 +57,48 @@ multiple input sections:
 IMPORTANT: Make sure a file is not defined more than once across all inputs
 because this can lead to unexpected behaviour.
 
+[[file-identity]]
+==== Reading files on network shares and cloud providers
```

Comment:
Side note: if we want to double down on network share support, then we also need to improve the file reader to reset to an older offset if an `n` 0-byte sequence has been read and/or call Stat once we reach an already known file offset. e.g. the read would call 'Stat' to get the file size and only read up to `sz`. Once we reach `sz` we call `Stat` again to update `sz` and continue reading until the offset again reaches `sz`. The call to `Stat` and/or resetting to an older offset helps to workaround races in some network share protocols by forcing explicit file metadata updates.

Reply:
Follow up issue: https://github.com/elastic/beats/issues/19736